### PR TITLE
Make Guzzle 5 a suggestion not a requirement

### PR DESCRIPTION
**Pull request by richardmiller**
Initial PR to make Guzzle 5 a suggestion rather than a requirement. Intending to follow up with a Guzzle 6 compatible implementation of the Feature Requester.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,17 +14,19 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "guzzlehttp/guzzle": "5.*",
-        "guzzlehttp/cache-subscriber": "0.1.*"
+        "php": ">=5.4"
```
Comment (jkodumal) [inline composer.json]: CI tests are failing-- it looks like the dependency changes require PHP 5.5.

**Comment by jkodumal** [comment]
Thanks for the contribution-- would be happy to merge this in once the CI tests pass!

**Comment by richardmiller** [comment]
Will take a look, not sure why moving the deps without changing the version constraints would have caused that. 

By the way php 5.4 was end of life a while ago, 5.5 will be in 22 days

**Comment by jkodumal** [comment]
Thanks-- no issues at all with bumping the requirements up to 5.5.

**Comment by richardmiller** [comment]
@jkodumal I've updated the php requirement, the version of PHP that circle ci uses and fixed the version one of the dev dependencies being pulled in as the latest version needed php >=5.6

**Comment by jkodumal** [comment]
Sorry for the delay @richardmiller !

**Pull request merged**